docs/API.md

```diff
@@ -0,0 +1,65 @@
+# API
+
+You can use `tslint-to-eslint-config` programmatically in your Node apps.
+It provides a **[`convertTSLintConfig`](#convertTSLintConfig)** function to find relevant configurations on disk and output the generated ESLint configuration.
+
+## `convertTSLintConfig`
```

Comment:
@JamesHenry is this API the kind of thing you're looking for?

If you're expecting the Angular toolchain to also convert comments and editor configs, we could also, say, export those individually, along with a general catch-all `convert` function to run all the conversions. Otherwise I'd default to filing that as a followup issue.

Reply:
Alternately, if some or all configs have already been loaded in, the relevant portions of `convertTSLintConfig` could be exported as well, to allow external tooling to piece it together...

Reply:
Discussed in a 1:1: we'll need to export the relevant portions of the API. The full `convertTSLintConfig` reads to and writes from the file system, but the nx tooling this PR is intended for is all in-memory. Excellent.